src/braket/ocean_plugin/braket_sampler.py

```diff
@@ -216,9 +216,9 @@ def sample_ising_quantum_task(
             h = dict((v, b) for v, b in enumerate(h) if b or v in self.nodelist)
 
         edges = self.edgelist
+        sorted_edges = tuple(sorted(set((u, v) if u < v else (v, u) for u, v in J)))
```

Comment:
Can this be extracted into a method and cached so it's only created once and used again below?

Reply:
NVM, these methods are entirely independent. You can extract a method if you want, but LGTM